.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+Thank you for your pull request. This repository has been archived.
+Please open the PR in https://github.com/nodejs/node/compare
```

Comment:
Went with https://github.com/nodejs/node/pulls for as much focusing as possible 🤷‍♂️